client/my-sites/site-settings/navigation-back-button/README.md

```diff
@@ -0,0 +1,41 @@
+*Navigation Back Button*
+
+This component is used to display a `Back` button for redirection to the previously
+visited site.
+
+#Usage:
+
+```js
+import NavigationBackButton from 'my-sites/site-settings/navigation-back-button';
+...
+
+getRoute() {
+	... // return a route
+}
+
+<span className="....">
+		<NavigationBackButton redirectRoute={ this.getRoute() } { ...this.props } />
+</span>
+
+```
+
+#Props (required):
+
+- `redirectRoute` -- destination redirect route (string)
+
+##Exemplary use
```

Comment:
I'd say we need only the section that demonstrates how `NavigationBackButton` works, without the one that demonstrates how `getRoute` works - to me that one is unnecessary in the context of the back button.